sdcm/sct_events/setup.py

```diff
@@ -144,5 +144,10 @@ def enable_default_filters(sct_config: SCTConfiguration):  # pylint: disable=unu
     DbEventsFilter(db_event=DatabaseLogEvent.BACKTRACE, line='Rate-limit: suppressed').publish()
     DbEventsFilter(db_event=DatabaseLogEvent.WARNING, line='abort_requested_exception').publish()
 
+    if SkipPerIssues('scylladb/scylla-enterprise#4691', params=sct_config):
```

Comment:
I'm not sure it should be based on this issue, if it agreed there it's acceptable, the issue would be closed.

And it doesn't seem like there's an agreement there to ever fix this error print.

So for now let's leave a reference to the discussion in a comment in the code.

Reply:
Updated